src/index.js

```diff
@@ -28,6 +28,7 @@ const randomMsg = require('./helpers').randomMessage;
 
 
 exports.handler = function (event, context, callback) {
+    console.log(event)
```

Comment:
Yes, we got fed up with having to add logging in every time we wanted to know what was going on :)